extensions/amp-mega-menu/0.1/test/test-amp-mega-menu.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-mega-menu';
+import {Keys} from '../../../../src/utils/key-codes';
+import {toggleExperiment} from '../../../../src/experiments';
+import {tryFocus} from '../../../../src/dom';
+
+describes.realWin(
+  'amp-mega-menu',
+  {
+    amp: {
+      extensions: ['amp-mega-menu'],
+    },
+  },
+  env => {
+    let win, doc, element;
+
+    beforeEach(() => {
+      win = env.win;
+      doc = win.document;
+
+      element = getAmpMegaMenu();
+      doc.body.appendChild(element);
+
+      toggleExperiment(win, 'amp-mega-menu', true);
```

Comment:
Let's add a TODO here to remove the experiment toggle code post launch. I fact, we should add a cleanup issue for each experiment we introduce and reference the issue number for cleanup actions in the code.